app/controllers/issue_templates_controller.rb

```diff
@@ -129,48 +126,52 @@ def set_pulldown
 
       if inherit_templates.any?
         inherit_templates.each do |x|
-          group.push([x.title, x.id, {:class => "inherited"}])
+          group.push([x.title, x.id, { class: 'inherited' }])
           if x.is_default == true
              if project_default_template.blank?
-              @default_template = x
+              default_template = x
             end
           end
         end
       end
     end
 
-    @globalIssueTemplates = GlobalIssueTemplate.joins(:projects).where(["tracker_id = ? AND projects.id = ?",
-                                                                        @tracker.id, @project.id]).order('position')
+    global_issue_templates = GlobalIssueTemplate.joins(:projects).where(['tracker_id = ? AND projects.id = ?',

```

Comment:
__[rubocop]__
* Line is too long. [110/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)